src/pkg/selectors/testdata/details.go

```diff
@@ -323,3 +325,21 @@ func GetDetailsSet() *details.Details {
 		},
 	}
 }
+
+func MakeErrors(failure, recovered, skipped bool) fault.Errors {
```

Comment:
this needs to be moved to a different package. Perhaps the `src/pkg/fault` package?

Reply:
moved both: errors to fault, details to details.